presto-main-base/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -450,6 +453,18 @@ public void checkCanAddConstraint(Identity identity, AccessControlContext contex
         }
     }
 
+    @Override
+    public List<ViewExpression> getRowFilters(Identity identity, AccessControlContext context, CatalogSchemaTableName tableName)
+    {
+        return Collections.emptyList();
```

Comment:
May be these can be loaded in the same way other checks are loaded : https://github.com/prestodb/presto/pull/24277/files#diff-05d844acdb3f42de1fbc27315e05243462acea86342d988ade68384ca5a3a094R149

Reply:
I'm not seeing anything from your link and not quite sure what you're referring to, what other checks?